Orange/data/domain.py

```diff
@@ -227,16 +182,22 @@ def __getitem__(self, idx):
             return self.metas[-1-idx]
 
     def __contains__(self, item):
-        """
-        Return `True` if the item (`str`, `int`, :class:`Variable`) is
-        in the domain.
+        """Check whether this item is in the domain.
+
+        Parameters
+        ----------
+        item : str, Variable, int
+            The variable name, descriptor or index.
+
+        Returns
+        -------
+        bool
+            Whether the item exists in the domain.
         """
         return item in self._indices
 
     def __iter__(self):
-        """
-        Return an iterator through variables (features and class attributes).
-        """
+        """Return an iterator through variables (features and class attributes)."""
```

Comment:
Add

> TODO: Make domain iterate over whole domain (including metas).

We had this in the works before, it just didn't pan out. Should be easier now.